docs/javascript.md

```diff
@@ -88,6 +88,32 @@ Takes these options:
 
 Full documentation on how to design multivariate tests, use the data in GA and construct hypothesis tests is on its way soon.
 
+
+## Reporting to Google Content Experiments
+The toolkit includes a library for multivariate testing that is capable of reporting data into Google Content Experiments. 
+
+### To create a new experiment
+
+1. Log in to Google Universal Analytics.
+2. Follow [these instructions](https://support.google.com/analytics/answer/1745152?hl=en-GB) to set up your experiment.
```

Comment:
It'd be worth mentioning which permissions in Google Analytics are necessary to be able to do this, as well as who can grant them (ie the Performance Analytics team)